File: VideoLibrary/src/main/java/com/skyworth/videolibrary/ui/VideoActivity.java
package com.skyworth.videolibrary.ui;

import androidx.appcompat.app.AppCompatActivity;
import android.os.Bundle;

import com.alibaba.android.arouter.facade.annotation.Route;
import com.skyworth.baselibrary.utils.ARouterPathConstant;
import com.skyworth.videolibrary.R;

@Route(path = ARouterPathConstant.VideoMainActivity)
public class VideoActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_video);
    }
}

File: VideoLibrary/src/main/java/com/skyworth/videolibrary/ui/VideoActivity.kt
package com.skyworth.videolibrary.ui

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.alibaba.android.arouter.facade.annotation.Route
import com.skyworth.baselibrary.utils.ARouterPathConstant
import com.skyworth.videolibrary.R

@Route(path = ARouterPathConstant.VideoMainActivity)
class VideoActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_video)
    }
}